learnEcl/1200-dataset.md

```diff
@@ -102,7 +102,7 @@ OUTPUT(SalaryAvg_DS, NAMED('SalaryAvg_DS'));
 
 ## Logical File
 
-A sprayed file on cluster. HPCC Systems supports different file formats such as XML, JSON, THOR, and CSV.
+A sprayed file on the cluster. HPCC Systems supports different file formats such as XML, JSON, THOR, and CSV.
```

Comment:
Logical files are any files saved on the cluster, not just sprayed files.  They are "logical" because a single name (the path) refers to all the data even if that data is distributed among many storage locations.